packages/quantic/cypress/e2e/default-1/refine-modal-content/refine-modal-content-expectations.ts

```diff
@@ -263,9 +264,50 @@ function refineContentExpectations(selector: RefineContentSelector) {
         })
         .logDetail('should order the facets correctly');
     },
+
+    refineSortOptionsEqual: (options: {value: string; label: string}[]) => {
+      options.forEach((option) => {
+        selector
+          .sortOption(option.value)
+          .should('exist')
+          .should('contain', option.label)
+          .logDetail(
+            'should contain the sort option ${option.label} in the refine modal sort component'
+          );
+      });
```

Comment:
What's the difference between this expectation and the expectation `sortOptionsEqual ` on line 291? they are both using the same selectors?

The expectation should select the sort component in the refine modal and the other expectation should select the sort outside the the refine modal, but the selector currently used in the same for both so how are you doing that?

Reply:
its using the same selector but not the same root. The 2 expectations use different selectors `RefineContentSelector` & `SortSelector` so they are not targetting the same thing. I did essentially the same thing we do in the facets.

Reply:
changed it to different selectors, they are now clearly for either refine modal or basic sort on the page